Warfare/Warfare/Map.swift

```diff
@@ -156,7 +156,7 @@ class Map: SKNode {
 
     func getVillage(region: [Tile]) -> Village? {
```

Comment:
When you invade a region you can split it into a region where the owner isn't in it.
This checks if the owner is there or not.